sql/core/src/main/scala/org/apache/spark/sql/execution/arrow/ArrowUtils.scala

```diff
@@ -120,4 +121,19 @@ object ArrowUtils {
       StructField(field.getName, dt, field.isNullable)
     })
   }
+
+  /** Return Map with conf settings to be used in ArrowPythonRunner */
+  def getPythonRunnerConfMap(conf: SQLConf): Map[String, String] = {
```

Comment:
Maybe move this function out of `ArrowUtils`? Doesn't seem to be Arrow specific.